spark/pyspark/arctern_pyspark/plot.py

```diff
@@ -0,0 +1,128 @@
+            dict_collect['points']=[]
+        dict_collect['points'].append(geo_dict['coordinates'])
+    elif geo_dict['type']=='MultiPoint':
+        if 'points' not in dict_collect:
+            dict_collect['points']=[]
+        dict_collect['points'].extend(geo_dict['coordinates'])
+    else:
+        from osgeo import ogr
+        geo = ogr.CreateGeometryFromJson(json.dumps(geo_dict))
+        if geo.HasCurveGeometry():
+            geo = geo.GetLinearGeometry()
+            geo = geo.ExportToJson()
+            geo = json.loads(geo)
+            _flat_geoms(geo,dict_collect)
+        else:
+            raise RuntimeError(f"unsupported geometry: {geo_dict}")
+            
+def _plot_collection(ax, plot_collect):
+    if(len(plot_collect)==0):
+        return None
+    
+    try:
+        from descartes.patch import PolygonPatch
+    except ImportError:
+        raise ImportError(
+            "The descartes package is required for plotting polygons in geopandas. "
+            "You can install it using 'conda install -c conda-forge descartes' "
+        )
+    try:
+        import matplotlib
+        from matplotlib.collections import PatchCollection,LineCollection
+        from matplotlib.colors import is_color_like
+    except ImportError:
+        raise ImportError(
+            "The matplotlib package is required for plotting polygons in geopandas. "
+            "You can install it using 'conda install -c conda-forge descartes' "
+        )
+    
+        
+    if 'polygons' in plot_collect:
+        collection = PatchCollection([PolygonPatch(geo) for geo in plot_collect['polygons']])
+        ax.add_collection(collection, autolim=True)
+    if 'lines' in plot_collect:
+        collection = LineCollection(plot_collect['lines'])
+        ax.add_collection(collection, autolim=True)
+    if 'points' in plot_collect:
+        x = [p[0] for p in plot_collect['points']]
+        y = [p[1] for p in plot_collect['points']]
+        collection = ax.scatter(x,y)
+    ax.autoscale_view()            
+
+def _plot_spark_data_frame(ax, geoms):
+    import json
+    from pyspark.sql.functions import col
+    
+    if len(geoms.columns) != 1:
+        raise RuntimeError(f"geoms should has only one colums, input schema = {geoms.schema}")
+    head_row = geoms.head()
+    if head_row==None:
+        return ax
+    head_row = head_row[0]
+    if isinstance(head_row, str): # input is geojson
+        geoms = geoms.collect()
+    elif isinstance(head_row,bytearray): #input is wkb
+        from arctern_pyspark._wrapper_func import ST_AsGeoJSON
+        geoms = geoms.select(ST_AsGeoJSON(col(geoms.columns[0])))
+        geoms = geoms.collect()
+    else:
+        raise RuntimeError(f"unexpect input type, {type(head_row)}")
+        
+    plot_collect = dict()
+    for geo in geoms:
+        geo_dict = json.loads(geo[0])
+        _flat_geoms(geo_dict,plot_collect)
+    
+    _plot_collection(ax, plot_collect)
+
+
+def plot(ax, geoms):
```

Comment:
Since we can only plot one column, why not passing a series instead of a dataframe?